modules/arm_plugin/scripts/arm_cpu_plugin_build.sh

```diff
@@ -145,24 +150,39 @@ ARCHDIR=`ls $OPENVINO_HOME/bin` && \
 cd $DEV_HOME || fail 12 "OpenVINO build failed. Stopping"
 
 #OpenVINO python
+# prepare cross-enviroment
+/usr/bin/python3 -m crossenv /opt/python3.7_arm/bin/python3.7m /tmp/cross_venv
+. /tmp/cross_venv/bin/activate && \
+cross-python -m pip install wheel && \
+cross-python -m pip install numpy==1.20.0
+
+# copy licencing directory
+cp -rf $OPENVINO_HOME/licensing $OPENVINO_HOME/src/bindings/python/
+
 [ "$UPDATE_SOURCES" = "clean" -a -e $OPENVINO_HOME/pbuild ] && rm -rf $OPENVINO_HOME/pbuild
 mkdir -p $OPENVINO_HOME/pbuild && \
 cd $OPENVINO_HOME/pbuild && \
 cmake -DInferenceEngineDeveloperPackage_DIR=$OPENVINO_HOME/build \
-      -DENABLE_PYTHON=ON -DPYTHON_EXECUTABLE="/opt/python3.7_arm/bin/python3.7m" \
+      -DOV_CPACK_RUNTIMEDIR="/arm_cpu_plugin/armcpu_package/runtime/lib/aarch64" \
+      -DENABLE_PYTHON=ON -DPYTHON_EXECUTABLE=`which cross-python` \
+      -DENABLE_WHEEL=ON \
       -DPYTHON_INCLUDE_DIRS=/opt/python3.7_arm/include/python3.7m \
       -DPYTHON_LIBRARIES=/opt/python3.7_arm/lib \
+      -DPYTHON_LIBRARY=/opt/python3.7_arm/lib/libpython3.7m.so \
       -DPYTHON_MODULE_EXTENSION=".so" \
       -DPYBIND11_FINDPYTHON=OFF \
       -DPYBIND11_NOPYTHON=OFF \
       -DPYTHONLIBS_FOUND=TRUE \
       -DCMAKE_BUILD_TYPE=$BUILD_TYPE -DENABLE_DATA=OFF \
       -DCMAKE_EXE_LINKER_FLAGS=-Wl,-rpath-link,$STAGING_DIR/opencv/lib \
       -DCMAKE_TOOLCHAIN_FILE="$OPENVINO_HOME/cmake/$TOOLCHAIN_DEFS" \
+      -DCMAKE_INSTALL_PREFIX=$STAGING_DIR \
+      -DCMAKE_INSTALL_LIBDIR="/arm_cpu_plugin/armcpu_package/runtime/lib/aarch64" \
```

Comment:
I suppose it's fixed by it is fixed by PR https://github.com/openvinotoolkit/openvino/pull/13197?

Reply:
redundant flags has been removed